flask_api/tests/test_mediatypes.py

```diff
@@ -6,7 +6,7 @@
 
 class MediaTypeParsingTests(unittest.TestCase):
     def test_media_type_with_params(self):
-        media = MediaType('application/xml; schema=foobar, q=0.5')
+        media = MediaType('application/xml; schema=foobar; q=0.5')
```

Comment:
Thanks @james-emerton appreciate that.
Apologies for not being active on Flask API at the moment - I'd love to get back to it, but a case of focusing on one thing at a time right now.